core/external_list.py

```diff
@@ -4,134 +4,18 @@
 from sqlalchemy import or_
 from sqlalchemy.orm.session import Session
 
-from .metadata_layer import CSVMetadataImporter, ReplacementPolicy
+from .metadata_layer import ReplacementPolicy
 from .model import (
     Classification,
-    CustomList,
     CustomListEntry,
-    DataSource,
     Edition,
     Identifier,
     Subject,
-    Work,
     get_one_or_create,
 )
 from .util.datetime_helpers import utc_now
 
 
-class CustomListFromCSV(CSVMetadataImporter):
```

Comment:
Was this poor chap just collateral damage?

Reply:
This got removed because it was only used by the `UpdateStaffPicksScript`. So it seemed good to remove it. I should have annotated that when I submitted the PR since its not at all clear why I removed these. Thanks for checking on them.